src/main/java/firebase/sdk/dashboard/dao/UserDaoDatastore.java

```diff
@@ -0,0 +1,76 @@
+package firebase.sdk.dashboard.dao;
+
+import firebase.sdk.dashboard.data.SDK;
+import firebase.sdk.dashboard.data.SDKReleaseMetadata;
+import firebase.sdk.dashboard.data.VersionMetadata;
+import firebase.sdk.dashboard.data.Release;
+import firebase.sdk.dashboard.data.Platform;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.time.Instant;
+import java.io.IOException;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.CompositeFilter;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.QueryResultIterable;
+
+/* Class to implement UserDao */
+
+public class UserDaoDatastore implements UserDao {
+
+    public static final DatastoreService DATASTORE = DatastoreServiceFactory.getDatastoreService();
+
+    // Get a user from the datastore
+    public User getUser(String id){
+         FilterPredicate userPropertyFilter = makePropertyFilter("user", id;
+        Query query = new Query("User").setFilter(userPropertyFilter);
+
+        PreparedQuery preparedQuery = DATASTORE.prepare(query);
+        FetchOptions fetchOptions = FetchOptions.Builder.withDefaults();
+        QueryResultIterable<Entity> users = preparedQuery.asQueryResultIterable();
```

Comment:
nit: since you are only searching for a single user, you could use asSingleEntity() isntead of asQueryResultIterable(). That way you don't have to do a for loop